libnetwork/datastore/datastore.go

```diff
@@ -93,6 +92,7 @@ func DefaultScope(dataDir string) ScopeCfg {
 			Config: &store.Config{
 				Bucket:            "libnetwork",
 				ConnectionTimeout: time.Minute,
+				PersistConnection: true,
```

Comment:
Do you plan to open a follow-up PR to remove this option?

Reply:
Eventually, I guess. Do you want to do it?

Reply:
- https://github.com/moby/moby/pull/47302